src/properties.ts

```diff
@@ -0,0 +1,219 @@
+/**
+ * Property field types
+ * examples are propvided for the different types:
+ *
+ */
+export enum FieldTypes {
+  /**
+   * userName: {
+   *   type: csf.FieldTypes.TEXT,
+   *   label: 'Name',
+   *   defaultValue: 'Storyteller',
```

Comment:
While we're simplifying why not just make this 'value'?

Reply:
`initialValue` or `value`